examples/complete/versions.tf

```
@@ -1,7 +1,10 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = ">= 0.12.0, < 0.14.0"
```

I'm guessing we'll need to bump this version as well...

```suggestion
required_version = ">= 0.13.0, < 0.14.0"
```